### ISSUE TITLE
Fiber type grouping

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6766]
**Preferred term label:**
Fiber type grouping

**Synonyms**
Fibre type grouping

**Definition (free text, please give PubMed ID)**
An abnormal distribution of muscle fiber types in muscle tissue. Human skeletal muscle contains at least two fiber types recognizable by histochemical techniques. In transverse sections of normal skeletal muscle, type 1 and type 2 fibers are distributed in a random fashion. Grouping of fibers of the same type can be seen in certain peripheral neuropathies, thought to be due to reinnervation of denervated muscle fibers by sprouting axons. With grouping, motor units enlarge. The fibers of a motor unit, which are normally scattered, come to lie adjacent to one another. Histochemical examination shows groups of muscle fibers of the same histochemical type.

**Parent term (use hpo.jax.org/app)**
Abnormal muscle fiber morphology HP:0004303

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:500013 
PMID:32715519  4/4

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
